packages/super-components/components/msc-advices/src/msc-advices/msc-advices.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+  Component,
+  Prop,
+  h,
+  Element,
+  Host,
+  State,
+  Watch,
+  Fragment,
+} from '@stencil/core';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components/icon';
+import { ODS_BUTTON_VARIANT } from '@ovhcloud/ods-components/button';
+import { ODS_TILE_VARIANT } from '@ovhcloud/ods-components/tile';
+import { HTMLStencilElement } from '@stencil/core/internal';
+import { Locale, defaultLocale } from '@ovhcloud/msc-utils';
+import { getTranslations } from './translations';
+import { AdviceGroups, Translations } from './msc-advices.types';
+
+export interface IMscAdvices {
+  serviceType: string;
+  serviceName: string;
+  locale?: Locale;
+}
+
+@Component({
+  tag: 'msc-advices',
+  styleUrl: 'msc-advices.scss',
+  shadow: true,
+})
+export class MscAdvices implements IMscAdvices {
+  @Element() host!: HTMLStencilElement;
+
+  @Prop() public locale = defaultLocale;
+
+  @Prop() public serviceType = '';
+
+  @Prop() public serviceName = '';
+
+  @Prop() public dataTrackingPrefix = '';
```

Comment:
Missing in IMscAdvices

Reply:
dont need a prefix 'data::data' one attributes is enough

Reply:
Done